docs/content/docs/dev/table/materialized-table/statements.md

```diff
@@ -391,6 +392,101 @@ ALTER MATERIALIZED TABLE my_materialized_table
 - Schema evolution currently only supports adding `nullable` columns to the end of the original table's schema.
 - In continuous mode, the new refresh job will not restore from the state of the original refresh job. This may result in temporary data duplication or loss.
 
+# CREATE OR ALTER MATERIALIZED TABLE
+
+```
+CREATE OR ALTER MATERIALIZED TABLE [catalog_name.][db_name.]table_name
```

Comment:
I am curious what you think of the idea to change the existing CREATE statement docs to
`CREATE [OR ALTER]` in line with flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

and add specific text around the OR ALTER capability, in this way it is obvious that the CREATE options and the CREATE OR ALTER options have common options.